packages/dashboard-frontend/src/services/workspace-client/index.ts

```diff
@@ -32,54 +31,36 @@ export abstract class WorkspaceClient {
     // workspaceClientLib axios interceptor
     this.axios.interceptors.response.use(
       async response => {
-        if (
-          this.checkPathPrefix(response, '/api/') &&
-          (isUnauthorized(response) || isForbidden(response))
-        ) {
-          await this.deauthorize();
+        const isApi = this.checkPathPrefix(response, '/api/');
+        if (isApi) {
+          if (isUnauthorized(response)) {
+            signIn();
+          } else if (isForbidden(response)) {
+            signOut();
+          }
```

Comment:
@olexii4 got a bit lost why do we sign in on 401 and sign out on 403?

Reply:
It was a part of the old implementation. I have fixed it.